.drone.star

```diff
@@ -500,14 +501,14 @@ def uiTests(ctx):
 
     filterTags = params["filterTags"]
 
-    if ("full-ci" in ctx.build.title.lower()):
+    if ("full-ci" in ctx.build.title.lower() or ctx.build.event == "tag"):
```

Comment:
This looks like the only change that is required. If the event is a tag, then the full CI will run.
The information that is available about the build is in this object:
https://github.com/drone/drone/blob/master/plugin/converter/starlark/args.go

If one of those items has the tag name in it, then we can also test that the tag name starts with "v". But if not, then it is not really a problem - we can run full CI on every tag (all the tags on `master` should start with "v" anyway)

Reply:
yes sure @phil-davis,
also the trigger has some regex like with `v*`:
```
"trigger": {
            "ref": [
                "refs/heads/master",
                "refs/tags/v*",
                "refs/pull/**",
            ],
        },
```,
maybe tat also filters the tag